pkgs/tools/games/opentracker/default.nix

```diff
@@ -60,6 +60,6 @@ buildDotnetModule rec {
     license = licenses.mit;
     maintainers = [ maintainers.ivar ];
     mainProgram = "OpenTracker";
-    platforms = platforms.linux;
+    platforms = [ "x86_64-linux" ];
```

Comment:
Shouldn't the platform remain as is and just have `broken = (stdenv.isLinux && (!stdenv.isx86_64));`?

Reply:
I changed this because it seems like some dependencies aren't available for aarch64, so it's unlikely that there's upstream support for that.